iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/AreaInterchangeTargetModification.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2024, Coreso SA (https://www.coreso.eu/) and TSCNET Services GmbH (https://www.tscnet.eu/)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.modification;
+
+import com.powsybl.commons.report.ReportNode;
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.iidm.modification.topology.NamingStrategy;
+import com.powsybl.iidm.network.Area;
+import com.powsybl.iidm.network.Network;
+
+import java.util.Objects;
+
+/**
+ * Simple {@link NetworkModification} for an area interchange target update.
+ *
+ * @author Bertrand Rix {@literal <bertrand.rix at artelys.com>}
+ */
+public class AreaInterchangeTargetModification extends AbstractNetworkModification {
+
+    private final String areaId;
+
+    private final double interchangeTarget;
+
+    public AreaInterchangeTargetModification(String areadId, double interchangeTarget) {
+        this.areaId = Objects.requireNonNull(areadId);
+        this.interchangeTarget = interchangeTarget;
+    }
+
+    @Override
+    public String getName() {
+        return "AreaInterchangeModification";
+    }
+
+    @Override
+    public void apply(Network network, NamingStrategy namingStrategy, boolean throwException, ComputationManager computationManager, ReportNode reportNode) {
+        Area area = network.getArea(areaId);
+        if (area == null) {
+            logOrThrow(throwException, "Area '" + areaId + "' not found");
+            return;
+        }
+        area.setInterchangeTarget(interchangeTarget);
+    }
+
+    @Override
+    public NetworkModificationImpact hasImpactOnNetwork(Network network) {
+        impact = DEFAULT_IMPACT;
+        Area area = network.getArea(areaId);
+        if (area == null) {
+            impact = NetworkModificationImpact.CANNOT_BE_APPLIED;
+        } else if (area.getInterchangeTarget().isPresent() && Math.abs(interchangeTarget - area.getInterchangeTarget().getAsDouble())
+            < EPSILON || area.getInterchangeTarget().isEmpty() && Double.isNaN(interchangeTarget)) {
+            impact = NetworkModificationImpact.NO_IMPACT_ON_NETWORK;
+        }
```

Comment:
To improve readability, I suggest this formatting:
```suggestion
        } else if (area.getInterchangeTarget().isPresent()
                    && Math.abs(interchangeTarget - area.getInterchangeTarget().getAsDouble()) < EPSILON
                || area.getInterchangeTarget().isEmpty() && Double.isNaN(interchangeTarget)) {
            impact = NetworkModificationImpact.NO_IMPACT_ON_NETWORK;
        }
```

Reply:
Well, hard to see the difference because of github formatting but I'll try something...

Reply:
You can copy-paste the code if you want to see the suggestion.
But basically the suggested changes were to insert:
- new lines before the first `&&` and the first `||` in order to have the whole comparison to `EPSILON` on the same line;
- 4 additional spaces before the first `&&`, so it has not the same indent as the next line.

Reply:
But I saw your modification, it looks fine to me. Thanks!